test.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<meta charset="UTF-8" />
+		<meta name="viewport" content="width=device-width" />
+		<title>Test</title>
+	</head>
+	<body>
+		<script src="dist/get-video-id.browser.js"></script>
```

Comment:
The `src` used here should be the same `src` that the end users would expect to use (to help us catch any regressions in the build process). And it seems like we are not expecting the users to reach into the `dist/` to grab the files.

Reply:
Updated the docs.